spec/cc/analyzer/issue_spec.rb

```diff
@@ -3,20 +3,7 @@
 module CC::Analyzer
   describe Issue do
     let(:output) do
-      {
```

Comment:
This data was already basically identical to the contents of `Factory#sample_issue`, may as well use it instead.